packages/web3-validator/src/formats/number.ts

```diff
@@ -0,0 +1,6 @@
+import { FormatDefinition } from 'ajv';
+import { isUInt, isNumber } from '../validation/numbers';
+
+export const uint: FormatDefinition<string> = { validate: (data: string) => isUInt(data) };
+export const int: FormatDefinition<string> = { validate: (data: string) => isUInt(data) };
+export const number: FormatDefinition<string> = { validate: (data: string) => isNumber(data) };
```

Comment:
For all these validator methods (in this file and related), should `data` be `unknown`?

Reply:
No we can't. These are `formats` which applies to a particular `type`.  And the type is defined by the generic. Ajv will only allow these `formats` to be used with defined type. Otherwise will throw error.